website/src/pages/mapper/dev_guide/configuration.mdx

```diff
@@ -0,0 +1,367 @@
+import { Callout, Tabs, Table, Td, Th, Tr, Code } from "nextra/components";
+
+# Compose Stack Configuration
```

Comment:
The configuration is not specific to the compose stack, it'd be easier just to make most of this page reusable across everything.

As in - if you want to configure Carrot, whether it is local, azure, on-prem.